### PR TITLE
Add check of doms

### PR DESCRIPTION
**Pull request by rshym**
Add check that domains exist in deploy path.
This check is needed as protection against possible misuses.
Misuse 1. If specified deploy path is higher than actually needed, then script will find all kernel images and write them to dom0.
Misuse 2. If deploy path has no domu, script will quit and will left loop device hanging.

Also two minor fixes for misprint and default image size.

### CODE REVIEW

File: doc/mk_sdcard_image.sh [hunk 1]
```diff
@@ -13,8 +13,8 @@ usage()
 	echo "`basename "$0"` <-p image-folder> <-d image-file> [-s image-size-gb] [-u dom0|domd|doma]"
 	echo "	-p image-folder	Base daily build folder where artifacts live"
 	echo "	-d image-file	Output image file or physical device"
-	echo "	-s image-size	Optional, image size in GB"
-	echo "  -u domain	Optional, unpack the domain specified"
+	echo "	-s image-size	Optional, image size in GiB"
```
Comment (andr2000) [inline doc/mk_sdcard_image.sh]: how is this change relevant to the rest of the changes in this commit?
Please send a dedicated clean up patch

File: doc/mk_sdcard_image.sh [hunk 2]
```diff
@@ -13,8 +13,8 @@ usage()
...
+	echo "	-u domain	Optional, unpack the domain specified"
```
Comment (andr2000) [inline doc/mk_sdcard_image.sh]: ditto

File: doc/mk_sdcard_image.sh [hunk 3]
```diff
@@ -43,7 +43,7 @@ inflate_image()
 		return 0
 	fi
 
-	echo "Inflating image file at $dev of size ${size_gb}GB"
+	echo "Inflating image file at $dev of size ${size_gb}GiB"
```
Comment (andr2000) [inline doc/mk_sdcard_image.sh]: ditto

File: doc/mk_sdcard_image.sh [hunk 4]
```diff
@@ -75,7 +75,7 @@ partition_image()
 
 	sudo parted -s $1 mkpart primary ext4 1MiB 257MiB || true
 	sudo parted -s $1 mkpart primary ext4 257MiB 2257MiB || true
-	sudo parted -s $1 mkpart primary 3415MiB 7838MiB || true
+	sudo parted -s $1 mkpart primary 2257MiB 6680MiB || true
```
Comment (andr2000) [inline doc/mk_sdcard_image.sh]: dedicated patch please

File: doc/mk_sdcard_image.sh [hunk 5]
```diff
@@ -443,5 +456,7 @@ else
 	make_image $ARG_DEPLOY_PATH $loop_dev_in $ARG_IMG_SIZE_GB
 fi
 
+print_step "Syncing"
```
Comment (andr2000) [inline doc/mk_sdcard_image.sh]: dedicated patch

**Comment by andr2000** [comment]
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`

File: doc/mk_sdcard_image.sh [hunk 6]
```diff
@@ -428,10 +424,27 @@ if [ -z "${ARG_DEPLOY_DEV}" ]; then
...
+# Check that deploy path contains dom0, domd and doma
+dom0_name=`ls ${ARG_DEPLOY_PATH} | grep dom0-image-thin` || true
+domd_name=`ls ${ARG_DEPLOY_PATH} | grep domd` || true
+doma_name=`ls ${ARG_DEPLOY_PATH} | grep android` || true
+if [ -z "$dom0_name" ]; then
+	echo "Error: deploy path has no dom0."
+	exit 2
+fi
+if [ -z "$domd_name" ]; then
+	echo "Error: dploy path has no domd."
```
[the remark this answers is not in the document]
Comment (rshym) [reply doc/mk_sdcard_image.sh]: Thanks, fixed.
